src/Apps/ArtAdvisor/03-Instruction-Experiments/App.tsx

```diff
@@ -51,21 +52,23 @@ export const App: FC = () => {
       <Spacer y={4} />
       <Text variant="lg-display">Profile Builder</Text>
       <Spacer y={4} />
-      {messages.map((message, index) => {
-        // Guard against trying to rendering messages that are just for memory, sush as function tool messages.
-        if (message.role === "tool" || message.content === null) {
-          return null
-        }
+      <Box>
+        {messages.map((message, index) => {
```

Comment:
`StackableBorderBox` needs to be [enclosed by a single parent](https://github.com/artsy/palette/blob/f4d0df0219980250af3deced02b9743b67824041/packages/palette/src/elements/StackableBorderBox/StackableBorderBox.tsx#L14-L15) in order for the bottom border to render correctly.